packages/gatsby-theme-store/src/components/Shelf/index.tsx

```diff
@@ -9,12 +9,14 @@ import ShelfPaginationDots from './PaginationDots'
 import ShelfTitle from './Title'
 import type { ProductSummary_ProductFragment } from '../ProductSummary/__generated__/ProductSummary_product.graphql'
 
+type Product = ProductSummary_ProductFragment | undefined | null
```

Comment:
isn't this the same as `Maybe<ProductSummary_ProductFragment>` ?

Reply:
I thought I couldn't use in this file 😅 gonna change